force-app/main/default/flows/Before_Public_Event_Delete.flow-meta.xml

```diff
@@ -1,6 +1,40 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@bmtemakina I believe this flow is related to Public Event object, is that correct ? if so please remove this metadata form this PR and create another PR for this change.